scripts/build/companion_libs/050-zlib.sh

```diff
@@ -84,6 +84,7 @@ do_zlib_backend() {
     CT_DoLog EXTRA "Configuring zlib"
 
     CT_DoExecLog CFG                                  \
+    CC="${host}-gcc"                                  \
```

Comment:
I reverted this to test it out.. "GMP for host" built with cc, not the version of gcc requested. zlib used x86_64-build_unknown-freebsd11.0-gcc correctly.